src/subsai/models/whisper_api_model.py

```diff
@@ -9,16 +9,29 @@
 
 import os
 import ffmpeg
+import tempfile
 from subsai.models.abstract_model import AbstractModel
 from subsai.utils import _load_config
 from openai import OpenAI
 from pysubs2 import SSAFile
+from pydub import AudioSegment
+
+TMPDIR = tempfile.gettempdir()
+
+def split_filename(filepath):
+    path, full_filename = os.path.split(filepath)
+    filename, ext = os.path.splitext(full_filename)
+    return path,filename,ext
+
+path,filename,ext = split_filename('/Users/luka/Desktop/y2mate.is - AGI Inches Closer 5 Key Quotes Altman Huang and The Most Interesting Year -fPzp_sdCf2Y-1080pp-1711573970.mp3')
```

Comment:
I think you forget this debugging line here :smile:

Reply:
indeed ooops!